logic/src/commonMain/kotlin/com/wire/kalium/logic/data/asset/AssetMapper.kt

```diff
@@ -0,0 +1,11 @@
+package com.wire.kalium.logic.data.asset
+
+import com.wire.kalium.network.api.model.Asset
+import com.wire.kalium.network.api.model.AssetMetadata
+import com.wire.kalium.network.api.model.AssetRetentionType
+
+fun UploadAssetMetadata.toApiModel(md5: String): AssetMetadata {
```

Comment:
it's not a bad idea to use extension functions for mapper classes, but we can't have anyway to mock it for Unit testing, so changing it to somthing like
```kotlin
interface AssetMapper {
    fun toApiModel(uploadAssetMetadata :UploadAssetMetadata): AssetMetadata
    fun toDomainModel(asset: Assrt): UploadAssetId
}

class AssetMapperImpl: AssetMapper {
    // override
}
```

Reply:
Ok makes sense, solved it